sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py

```diff
@@ -1310,24 +1332,6 @@ async def create_and_process_run(
 
         return run
 
-    @overload
```

Comment:
Was this extra overload?

Reply:
Yes, essentially, we have the next overload, which cover the first one
```python
# The one, which was removed
def create_stream(
        self, thread_id: str, body: JSON, *, content_type: str = "application/json", **kwargs: Any
    ) -> _models.AsyncAgentRunStream:
    ...
# The one we still have 
    @overload
    async def create_stream(
        self, thread_id: str, body: Union[JSON, IO[bytes]], *, content_type: str = "application/json", **kwargs: Any
    )
```